notebook/tests/selenium/utils.py

```diff
@@ -356,8 +356,8 @@ def shift(browser, k):
     trigger_keystrokes(browser, "shift-%s"%k)
 
 def ctrl(browser, k):
-    """Send key combination Ctrl+(k)"""
-    trigger_keystrokes(browser, "control-%s"%k)
+    """Send key combination Ctrl+(k) or Command+(k) for MacOS"""
```

Comment:
Are there any relevant shortcuts which are `Ctrl-something` even on Macs? In some of the frontend code we use 'cmdtrl' to refer to cmd/ctrl shortcuts. Possibly we should call this function that too, if it's not too big a pain to rename it now.

Reply:
Do you mean if there is some use on Mac's ctrl key that could be needed for the app? If so, there isn't anyone I could think of, since that key is used for terminal exclusive commands, any interaction with the Notebook should use the cmd key. And for the `cmdtrl`, your recommendation is to use that term on the comment line so it could match the naming style used in this project? if that's correct, I can make the update without problem 👍

Reply:
I was actually thinking rename the method from `ctrl` to `cmdtrl`.

Reply:
Alright, will do that 👍